### PR TITLE
simplify `open`

### PR DESCRIPTION
**Pull request by t-bltg**
While trying to fix `nightly`.
Seems like there is a missing `close` somewhere...

### CODE REVIEW

File: src/backends/web.jl [hunk 1]
```diff
@@ -25,22 +25,19 @@ embeddable_html(plt::AbstractPlot) = html_head(plt) * html_body(plt)
...
-    @warn "Unknown OS... cannot open browser window."
 end
 
 function write_temp_html(plt::AbstractPlot)
     html = standalone_html(plt; title = plt.attr[:window_title])
     filename = string(tempname(), ".html")
-    output = open(filename, "w")
-    write(output, html)
-    close(output)
+    write(filename, html)
```
Comment (BeastyBlacksmith) [inline src/backends/web.jl:40]: I am surprised this works without opening a file first

Comment (t-bltg) [reply src/backends/web.jl:40]: See https://docs.julialang.org/en/v1/base/io-network/#Base.write

And https://github.com/JuliaLang/julia/blob/0231c224a1ff25aba975a182b465e06de10f72e5/base/io.jl#L459

Comment (BeastyBlacksmith) [reply src/backends/web.jl:40]: Thats handy

Comment (t-bltg) [reply src/backends/web.jl:40]: Yup, as convenient as `read(filename, String)`.

**Comment by BeastyBlacksmith** [comment]
> Seems like there is a missing `close` somewhere...

Are you sure? It looks kore like a bug in Pkg to me.

**Comment by t-bltg** [comment]
I mean by that on `nightly` we hang waiting for some `i/o` operations, I vaguely recall seeing something similar in some discussion in the julia repo (but don't recall which one ...).
```
[pid 8003] waiting for IO to finish:
 TYPE[FD/PID]       @UV_HANDLE_T->DATA
 pipe[18]           @0x29f8660->0x7f1664428880
 pipe[12]           @0x307d3b0->0x7f1664428920
 process[8015]      @0x2be3af0->0x7f1664c696d0
```

Probably minor (likely to be fixed by future commits on `julia@master`), but let's keep an eye on this.

**Pull request merged**